src/dftbp/dftb/extcharges.F90

```diff
@@ -25,7 +25,7 @@ module dftbp_dftb_extcharges
     private
 
     !> Number of point charges
-    integer :: nChrg
+    integer, public :: nChrg
```

Comment:
It is safer to return this value via a function instead of making it a public variable (otherwise it can be changed outside the module). The function should be a procedure for the type, so caries the same allocation status etc. as the instance of the type.

Reply:
I made the requested changes. The allocation was originally added because there was a segmentation fault and valgrind identified the stack allocation as the culprit, but I think that report was merely an artifact of having written pass the end of the arrays elsewhere. Now that the issue elsewhere has been resolved, the explicit allocation no longer appears to be necessary.